data/mods/Aftershock/maps/terrain_floraxeno.json

```
@@ -133,5 +133,87 @@
       "sound_fail": "squish.",
       "items": [ { "item": "withered", "prob": 50, "count": [ 1, 2 ] } ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_geo_mox",
+    "name": "moxphore bladder",
+    "description": "A large white coral like object grows around the rim of the geothermal vent, pulsating black veins riddle its skin, and shoot out from it.  While above it, red and purple tentacles caress a balloon-shaped organ as it breaths.",
+    "symbol": "^",
+    "color": "light_gray",
+    "move_cost_mod": -1,
+    "coverage": 65,
+    "required_str": -1,
+    "flags": [ "NOITEM", "MINEABLE", "BASHABLE", "BLOCK_WIND", "EMITTER" ],
+    "emissions": [ "emit_smoke_plume", "emit_hot_air_migo_seep" ],
+    "bash": {
+      "str_min": 65,
+      "str_max": 160,
+      "sound": "smash!",
+      "sound_fail": "thump.",
+      "items": [
+        { "item": "rock", "count": [ 10, 22 ] },
+        { "item": "sharp_rock", "count": [ 3, 7 ] },
+        { "item": "material_limestone", "charges": [ 2, 5 ], "prob": 30 },
+        { "item": "material_rocksalt", "count": [ 0, 1 ], "prob": 10 },
+        { "item": "material_rhodonite", "count": [ 0, 1 ], "prob": 1 },
+        { "item": "material_zincite", "count": [ 0, 5 ], "prob": 2 },
+        { "item": "alien_flesh", "count": [ 2, 5 ], "prob": 50 },
+        { "item": "alien_fat", "count": [ 1, 3 ], "prob": 30 },
+        { "item": "endochitin", "count": [ 2, 5 ], "prob": 30 },
+        { "item": "alien_lung", "count": [ 0, 2 ], "prob": 10 },
+        { "item": "alien_organs", "count": [ 1, 2 ], "prob": 5 },
+        { "item": "chitin_piece", "count": [ 2, 5 ], "prob": 10 }
+      ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_moxie_tube",
+    "name": "moxphore vein",
```

```suggestion
    "name": "breather",
```
I personally believe it would be best to just use Moxphore to refer to the actual creatures.  Mostly due to the fact that even if they are related in lore. they  rent that visually alike (which isnt a bad thing)